Docs: kill "Choosing a site" (#11464)
readthedocs/readthedocs.org: docs/user/index.rst

    |org_brand| hosts documentation for over 100,000 large
    and small open source projects.
    |com_brand| supports hundreds of organizations with product and internal documentation.

>     Learn more about these differences in `our pricing page <https://about.readthedocs.com/pricing/>`_.

Reviewer: ```suggestion
    Learn more about these differences on `our pricing page <https://about.readthedocs.com/pricing/>`_.
```

Also, "our pricing page" feels like an internal name. We could probably avoid this for something more descriptive.

Author: "the pricing page" ?

Author: ```suggestion
    Learn more about these differences on `the pricing page <https://about.readthedocs.com/pricing/>`_.
```

Reviewer: To clarify "pricing page" sounds the same to me as "socialaccount view" to me -- we know what it means, but users might not.

"Learn more about these differences on our plan comparison overview" is more descriptive.
